applications/GeoMechanicsApplication/custom_constitutive/truss_backbone_constitutive_law.cpp

```diff
@@ -0,0 +1,166 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//
+//  Main authors:    Wijtze Pieter Kikstra
+//
+
+// Project includes
+#include "truss_backbone_constitutive_law.h"
+#include "geo_mechanics_application_variables.h"
+#include "includes/properties.h"
+
+namespace Kratos
+{
+ConstitutiveLaw::Pointer TrussBackboneConstitutiveLaw::Clone() const
+{
+    return Kratos::make_shared<TrussBackboneConstitutiveLaw>(*this);
```

Comment:
Since this member function definition is contained in namespace `Kratos`, there is no need to repeat the namespace qualifier here:
```suggestion
    return make_shared<TrussBackboneConstitutiveLaw>(*this);
```

Reply:
Done.